bin/adam-submit

```diff
@@ -65,29 +63,14 @@ else
 fi
 
 # Split args into Spark args and ADAM args
-# NOTE: if Spark uses gatherSparkSubmitOpts in spark-submit, this is unnecessary
-function usage() {
-  echo "adam-submit <spark-args> <adam-args>"
-  exit 0
-}
-if [ -f "$SPARK_HOME"/libexec/bin/utils.sh ]; then
-  source "$SPARK_HOME"/libexec/bin/utils.sh
-else
-  source "$SPARK_HOME"/bin/utils.sh
-fi
-SUBMIT_USAGE_FUNCTION=usage
-gatherSparkSubmitOpts $NEW_OPTIONS
+SPARK_ARGS=${NEW_OPTIONS%% -- *}
+ADAM_ARGS=${NEW_OPTIONS##* -- }
```

Comment:
I'm not great with shell scripts.  Is this good enough?  Or is this crazy brittle?